### PR TITLE
[containers/*] Create a new workflow for verification

### PR DESCRIPTION
**Pull request by fmulero**
Signed-off-by: Fran Mulero <fmulero@vmware.com>

**Description of the change**

Add new github workflow using new Verification Engine actions for containers

**Benefits**
* Better code quality
* Reduce risk of including breaking changes
* Shorter review time

**Possible drawbacks**
N/A

**Applicable issues**
N/A

**Additional information**

### CODE REVIEW

File: .github/workflows/verify.yaml [hunk 1]
```diff
@@ -1,11 +1,6 @@
 name: 'VIB'
```
Comment (recena) [inline .github/workflows/verify.yaml:1]: Why did you change the name? We should follow the same approach as [here](https://github.com/bitnami/charts/tree/master/.github/workflows):

- `.github/workflows/vib-verify.yaml`: GitHub Pipeline to integrate with Content Platform and perform a verification.
- `.github/workflows/vib-publish.yaml`: GitHub Pipeline to integrate with Content Platform and perform a publishing.

Comment (fmulero) [reply .github/workflows/verify.yaml:1]: There was no reason to change it. Good catch

File: .vib/vib-verify.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (recena) [inline .vib/vib-verify.json:1]: @fmulero Could you review the indentation of this file? 2-spaces indentation.

Comment (fmulero) [reply .vib/vib-verify.json:1]: It should be fixed now

**Comment by fmulero** [comment]
Things that can be improved and known TODOs:
* Container tags are already set to `latest`, we should retrieve the real version in some way
* In the `vib-verify` jobs we are checking out the repository (around 200Mb) for every "flavor".  We only need the vib-verify.json file to run the container-lint and container-package action, so we could reduce the execution time downloading only that file and not the whole repository. We agreed to keep it simple at this point because we could need more resources from the repository in a near future

**Review by recena — APPROVED** [review]

File: .github/workflows/vib-verify.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+          fi
+      # Using actions/github-scripts because using exit 1 in the script above would not provide any output
+      # Source: https://github.community/t/no-output-on-process-completed-with-exit-code-1/123821/3
+      - id: show-error
+        name: Show error
+        if: ${{ steps.get-container.outputs.result == 'fail' }}
+        uses: actions/github-script@v3
+        with:
+          script: |
+              core.setFailed('${{ steps.get-container.outputs.error }}')
```
Comment (joancafom) [inline .github/workflows/vib-verify.yaml:63]: This will never be executed, as none of the cases above set `result::fail`. You may safely remove this block of code.
